conceptual/Aspects/Diagnostics.md

```diff
@@ -55,6 +57,11 @@ To suppress a diagnostic:
 
 The following logging aspect requires a `_logger` field to exist, but it is likely that this field will never be used in user code but only in generated code. Because the IDE does not see the generated code, it will report the `CS0169` warning, which is misleading and annoying to the user. The aspect suppresses this warning.
 
+<!--- and then to add insult to injury you go on to show me a way to surpress a warning I likely wouldn't have created given that a fabric is the obvious way to do logging.
```

Comment:
```suggestion
```

Reply:
You would use the fabric just to use the aspect, instead of adding it using a custom attribute, so this example is still relevant.